python/lsst/pipe/tasks/multiBandUtils.py

```diff
@@ -223,9 +224,11 @@ def makeIdFactory(self, dataRef):
         The actual parameters used in the IdFactory are provided by
         the butler (through the provided data reference.
         """
-        info = ExposureIdInfo(
-            int(dataRef.get(self.config.coaddName + datasetName)),
-            dataRef.get(self.config.coaddName + datasetName + "_bits")
-        )
+        try:
+            expId = getGen3CoaddExposureId(dataRef, coaddName=self.config.coaddName, includeBand=includeBand,
+                                           log=self.log)
+        except Exception:
```

Comment:
So, is this a gen3/gen switch?  or are we supposed to go back to `int(dataRef.get(self.config.coaddName + datasetName))` after gen2 goes away and we delete `getGen3CoaddExposureId`

Reply:
Oh I see, this is only used by Gen2 code.